go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/antihax/optional v1.0.0
+	github.com/golangci/golangci-lint v1.43.0
```

Comment:
The latest version is 1.46.1. I would suspect dependabot to immediately flag this.